yarn.lock

```diff
@@ -2908,21 +2895,21 @@
     debug "^4.3.4"
     typeorm "^0.3.10"
 
-"@sphereon/ssi-sdk-data-store-common@file:../ssi-sdk/packages/data-store-common/sphereon-ssi-sdk-data-store-common-v0.9.9-bram.tgz":
-  version "0.9.9-bram"
-  resolved "file:../ssi-sdk/packages/data-store-common/sphereon-ssi-sdk-data-store-common-v0.9.9-bram.tgz#c0a860d691fd76d7f3106013f47cca1d32152fbc"
+"@sphereon/ssi-sdk-data-store-common@file:../ssi-sdk/packages/data-store-common/sphereon-ssi-sdk-data-store-common-v0.9.15-bram.tgz":
+  version "0.9.15-bram"
+  resolved "file:../ssi-sdk/packages/data-store-common/sphereon-ssi-sdk-data-store-common-v0.9.15-bram.tgz#719cecc7b0b9f248eec1056bfa7b6d90f499cc2c"
   dependencies:
     "@veramo/core" "4.2.0"
     "@veramo/utils" "4.2.0"
     debug "^4.3.4"
     typeorm "^0.3.10"
 
-"@sphereon/ssi-sdk-data-store@file:../ssi-sdk/packages/data-store/sphereon-ssi-sdk-data-store-v0.9.9-bram.tgz":
-  version "0.9.9-bram"
-  resolved "file:../ssi-sdk/packages/data-store/sphereon-ssi-sdk-data-store-v0.9.9-bram.tgz#0c1e6ba239a5f0c45dce0f27bc4fff6bee601257"
+"@sphereon/ssi-sdk-data-store@file:../ssi-sdk/packages/data-store/sphereon-ssi-sdk-data-store-v0.9.15-bram.tgz":
```

Comment:
This will be problematic for other devs

Reply:
will be fixed once WAL-365 can be merged (which should have this fixed by then)